integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanReschedulingWithinCenterTest.java

```diff
@@ -320,7 +320,7 @@ public void testCenterReschedulingMultiTrancheLoansWithInterestRecalculationEnab
 
         // VERIFY THE INTEREST
         Float interestDue = (Float) ((HashMap) loanRepaymnetSchedule.get(2)).get("interestDue");
-        assertEquals("41.05", String.valueOf(interestDue));
+        assertEquals("45.41", String.valueOf(interestDue));
```

Comment:
@josehernandezfintecheandomx Are you sure it is valid to change the interest in this test case?

Reply:
It was set as original amount in the case